File: sdkwork-backend-sdk-flutter/lib/src/api/account_exchange_config.dart
import '../http/client.dart';
import '../models.dart';

class AccountExchangeConfigApi {
  final HttpClient _client;
  
  AccountExchangeConfigApi(this._client);

  /// Get points-to-cash rate
  Future<PlusApiResultAccountExchangeRateVO?> getPointsToCashRate() async {
    final response = await _client.get(ApiPaths.backendPath('/account/exchange-config/points-cash-rate'));
    return response is PlusApiResultAccountExchangeRateVO ? response : null;
  }

  /// Update points-to-cash rate
  Future<PlusApiResultAccountExchangeRateVO?> updatePointsToCashRate(AccountExchangeRateUpdateForm body) async {
    final response = await _client.put(ApiPaths.backendPath('/account/exchange-config/points-cash-rate'), body: body, contentType: 'application/json');
    return response is PlusApiResultAccountExchangeRateVO ? response : null;
  }
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/api/account_exchange_config.py
from typing import Any, Dict, List, Optional
from ..http_client import HttpClient
from ..models import AccountExchangeRateUpdateForm, PlusApiResultAccountExchangeRateVO

class AccountExchangeConfigApi:
    """account_exchange_config API client."""
    
    def __init__(self, client: HttpClient):
        self._client = client

    def get_points_to_cash_rate(self) -> PlusApiResultAccountExchangeRateVO:
        """Get points-to-cash rate"""
        return self._client.get(f"/backend/v3/api/account/exchange-config/points-cash-rate")

    def update_points_to_cash_rate(self, body: AccountExchangeRateUpdateForm) -> PlusApiResultAccountExchangeRateVO:
        """Update points-to-cash rate"""
        return self._client.put(f"/backend/v3/api/account/exchange-config/points-cash-rate", json=body)
